### PR TITLE
Build changes: Experiment with BrowserStack

### PR DESCRIPTION
**Pull request by margaree**

### CODE REVIEW

File: karma.browserstack.conf.js [hunk 1]
```diff
@@ -5,37 +5,51 @@ const merge = require('deepmerge');
 const defaultPattern = '+(components|helpers|mixins|templates)/**/*.test.js';
```
Comment (dlockhart) [inline karma.browserstack.conf.js:5]: From [the open-wc docs](https://open-wc.org/testing/testing-karma-bs.html) there's some helpers for using BrowserStack... but actually [looking at the code](https://github.com/open-wc/open-wc/blob/master/packages/testing-karma-bs/bs-settings.js) it doesn't really provide anything special, so nevermind!

File: components/inputs/test/input-text.test.js [hunk 2]
```diff
@@ -35,41 +35,6 @@ function pressEnter(elem) {
 
 describe('d2l-input-text', () => {
```
Comment (dlockhart) [inline components/inputs/test/input-text.test.js:37]: Were these changes intended to be in here?

Comment (margaree) [reply components/inputs/test/input-text.test.js:37]: Yup! I was overall trying to improve the build and these axe tests are also in a `.axe` file and were occasionally failing on edge. The change to `localize-mixin` was similar - that test was flaky as well (on other browsers too) for the `async` case and after bumping the timeout I have not seen it fail again.

File: .travis.yml [hunk 3]
```diff
@@ -32,15 +32,15 @@ jobs:
...
-            npm run test:sauce || travis_terminate 1;
+            npm run test:bs || travis_terminate 1;
         else
             echo "Not a pull request and/or no secure environment variables, running headless tests...";
             npm run test:headless || travis_terminate 1;
         fi
       env:
-        - SAUCE_USERNAME: Gaudi1
-        # SAUCE_ACCESS_KEY
-        - secure: bwPUP8HCJ4x/VQHYT83fw28MF6fZqcCFYYLQIuYl5d8QXPuHNiON+36YSEkIix/2AjuQR8OtNqy7MPem013QUzEl9ayJ+P/Kto3hN84D6fNj4DXKUtDywDeAs6B0hv3OZZIUJ2ixqClyGDRoK9M51NzS3d9Nts8rEDTJY5CqnD7s+0mOUXfmr7ehStQfs4JdsGApdp5ByA7btC6t3vNAUSoGOlnfKGb4VP/cCXAS64Suy/8Q09fCCKr2BMswQwR6hwb52EJ50W9JbpXu/S0DwrwwM5ZGCG4+SSXqrMoaWXeoOjjQ+tdTx+Rj/cvYO0IE9vJvEMubFCzjs57d2r9IwoKmJg27Fh7YPlZk4YEhcge2vDO9AmlGKGDKeEYco4fXelAmmCcFjmrWXjH8Uc31n2wZj4yYg6AKU2M+GvMYNmh+aI1qGJ+urpQigYdf8IWpqryW19wiOzRm+N6JhdFGlfgnJMEv/OmdTwGOXKA1Qlmc0Bv8ZjWHa6bVMrwnueygoUwc7rWYQg0SjIaW8zhTWwy2JkBoCD1/TpKYTNOdL4VSaxpRSCQn4uAlJ2rNBIyv7FM0S7ycKMTVQGXIyaRo28vKUEGUDYGYWbu++5fBxRfaHWeE79ZFE823LyFQetTUmr7ms/i4pqkaM/m+q840s34lLwmupSoUoVBAGezVsk8=
```
Comment (dlockhart) [inline .travis.yml:43]: Maybe keep these until we're ready to commit to switching? 🤷

File: karma.bs.conf.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+			browsers: Object.keys(customLaunchers),
+			reporters: ['dots', 'BrowserStack'],
+			singleRun: true,
+			browserDisconnectTimeout : 20000, // default 2000
+			browserDisconnectTolerance : 3, // default 0
+			browserNoActivityTimeout: 200000, // default 10000
+		}),
+	);
+	return config;
+};
```
Comment (dlockhart) [inline karma.bs.conf.js:76]: Did you meant to rename `karma.sauce.conf.js` -> `karma.sauce.js`?

Comment (margaree) [reply karma.bs.conf.js:76]: oops!

**Comment by margaree** [comment]
The tests are passing in BrowserStack but travis is saying they failed - I'll investigate more.
